benchmarks/cpp/heuristic_cache.cpp

```diff
@@ -107,8 +107,7 @@ static void NvFuserScheduler_LayerNormBackward_HeuristicCache(
   auto runtime = getLayerBackwardNormRuntime(
       std::move(fusion_ptr), executor_cache, aten_inputs, shape, norm_shape);
 
-  KernelArgumentHolder args =
-      KernelArgumentHolder::createKernelArgumentHolder(aten_inputs);
+  KernelArgumentHolder args = KernelArgumentHolder(aten_inputs);
```

Comment:
nit: might consider just
```c++
KernelArgumentHolder args(aten_inputs);
```
🤷nbd

Reply:
This is in a follow up PR: https://github.com/NVIDIA/Fuser/pull/3879